database/seedDatabase.js

```diff
@@ -1,5 +1,6 @@
 const mongoose = require('mongoose');
 const faker = require('faker');
+const Photo = require('./Model.js')
 
 mongoose.connect('mongodb://localhost/photos');
```

Comment:
Not sure you need mongoose.connect since it the connection is already opened in your database.index.js file